src/VscProcess.cpp

```diff
@@ -60,6 +60,11 @@ VscProcess::VscProcess() : myEStopState(0)
   else
   {
     ROS_INFO("Connected to VSC on %s : %i", serial_port_.c_str(), serial_speed_);
+    // enable VSC Remote Status, we only need to set this once 
+    // thus we can assume that by the time the mower leaves the manufacturer the VSC has run this once.
+    uint8_t enableMessage = 1;
+    uint16_t milliSecondInterval = 1000;
+    vsc_send_control_msg_rate(vscInterface, MSG_VSC_REMOTE_STATUS, enableMessage, milliSecondInterval);       
```

Comment:
Since the `vsc_send_control_msg_rate` message gets written to the VSC EEPROM, we need to be very careful how often we send this message. (EEPROM have very limited number of write/erase cycles). 
We might want to consider checking to see if the VSC is outputting the VSC_REMOTE_STATUS before sending this message. 
@AGummyBear what do you think. Is this something we should be concerned with?

Reply:
Hmm, I like that idea. 
So waiting 2 seconds for the first Remote Status message and checking that vscInterface is valid before trying to enable the message.

Reply:
I just did a quick google, and most of the results are specifically targeted towards Arduino, but they're saying 100k write/erase cycles as a specified life. Under those conditions,  I would not be worried about the EEPROM. A single write at each start of our system is still pretty low. There also may be safeguards within the system to know that if the requesting setting is the same as the current setting -> do nothing. That's a pretty big assumption though for us to make.

A lot of systems I've used before also have options to write settings locally only vs to EEPROM. We'd have to reconfigure at each boot like we do now, but it alleviates any concerns about wearing out the memory.

Finally though, I do like the idea of just checking to see that we're not receiving the message, and if we're not, then enable it.

That was a lot of talking to say probably just do what Evan suggested and only send this control message enable request if we're not receiving the status message already.

Reply:
@edavies64 @AGummyBear 
I tested the "enable only if not receiving messages" functionality by :

1. disabling the status message 
2. re-pulling the branch
3. ```catkin_make```
4. ```rosrun greenzie_bringup plaut```
5. Checking that `/safety/health_status` was publishing.

Reply:
@edavies64 @AGummyBear 
For some reason it takes 1 time running ```rosrun greenzie_bringup plaut``` then killing it and then ```rosrun greenzie_bringup plaut``` before the  status messages are sent.

With the prior set up I had it turned the messages on from the first start of plaut.